universities/organizations.json

```diff
@@ -449,6 +776,18 @@
     "formal_name": "University of Luxembourg",
     "country": "LUX"
   },
+  {
+    "id": "14148",
+    "icpc_id": "14148",
+    "name": "UoL",
```

Comment:
Duplicate with "University of Liverpool", but this data probably comes from ICPC so then it should be changed elsewhere :stuck_out_tongue:

Reply:
correct, short name is also not required to be unique.